ccpp/suites/suite_RRFS_parallel.xml

```diff
@@ -0,0 +1,90 @@
+<suite name="RRFS_parallel" lib="ccppphys" ver="4">
```

Comment:
Please do not add this suite definition file, it is not related to flake and there are several RRFS-related suite definition files already.